### PR TITLE
Testcoverage

### PR DESCRIPTION
**Pull request by arkwave**
Added in tests for test_cadence_features and test_transformation.py.
- coverage for cadence_features up to 92% from 78%
- coverage for transformation.py up top 83% from 68%

Added in line in cadence_features.py to account for python lists incorrectly being passed in as input.

### CODE REVIEW

**Comment by stefanv** [comment]
Please rebase on master to resolve conflict.

File: cesium/features/cadence_features.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
 
 
 def double_to_single_step(cads):
-    """Ratios (t[i+2] - t[i]) / (t[i+2] - t[i+1])."""
+    """Ratios t[i+2] - t[i] / (t[i+2] - t[i+1])."""
```
Comment (bnaul) [inline cesium/features/cadence_features.py]: This function actually was implemented incorrectly; if you rebase and just accept my changes from 808e0c7a4a63aa55164a7eb873c7a5e718283066 then you shouldn't need to touch this function.

**Comment by bnaul** [comment]
Can you also:
1) fix the line lengths and other PEP8 violations (the `flake8` or `pep8` utilities will show you the problems)
2) edit the title of this PR to be more informative?

File: cesium/features/tests/test_cadence_features.py [hunk 2]
```diff
@@ -43,3 +43,33 @@ def test_find_sorted_peaks():
 
     x = np.array([0,3,3,5,0]) # Tie is a peak only if greater than next value
     npt.assert_allclose(cf.find_sorted_peaks(x), np.array([[3,5]]))
+
+def test_peak_ratio():
+    """ Test peak ratio method."""
+    x = np.array([0,5,2,3,1])
+    peaks1 = cf.find_sorted_peaks(x)
+    npt.assert_almost_equal(cf.peak_ratio(peaks1,0,1), 5/3)
```
Comment (stefanv) [inline cesium/features/tests/test_cadence_features.py]: ^^^

**Comment by stefanv** [comment]
@bnaul Want to take a look?

File: cesium/tests/test_transformation.py [hunk 3]
```diff
@@ -27,3 +27,35 @@ def test_train_test_split():
     npt.assert_equal(sum(ts.target == 'class1' for ts in test), n_class1 / 2)
     npt.assert_equal(sum(ts.target == 'class2' for ts in train), 1 * n_class2 / 2)
     npt.assert_equal(sum(ts.target == 'class2' for ts in test), n_class2 / 2)
+
+def test_transform_ts_files():
+    n_class1 = 4
+    n_class2 = 8
+    transform_type = "Train/Test Split"
+    time_series = []
+    for i in range(n_class1):
```
Comment (bnaul) [inline cesium/tests/test_transformation.py]: This would be better as a list comprehension instead of a for loop.

**Comment by bnaul** [comment]
Still needs PEP8 fixes (at least for line lengths, some of which are super long). Otherwise looks good 👍 .

**Comment by bnaul** [comment]
:+1: @stefanv?

**Comment by stefanv** [comment]
Thanks, @arkwave !

**Pull request merged**